apps/webapp/app/v3/services/batchTriggerTask.server.ts

```diff
@@ -26,10 +26,9 @@ export class BatchTriggerTaskService extends BaseService {
       const existingBatch = options.idempotencyKey
         ? await this._prisma.batchTaskRun.findUnique({
             where: {
-              runtimeEnvironmentId_taskIdentifier_idempotencyKey: {
+              runtimeEnvironmentId_idempotencyKey: {
                 runtimeEnvironmentId: environment.id,
                 idempotencyKey: options.idempotencyKey,
-                taskIdentifier: taskId,
```

Comment:
`findUnique` with an include? Should we change this to a `findFirst` with the runtimeEnvironmentId and idempotencyKey in the where?

Reply:
I guess this was already there… I guess it's unlikely that this has been combined by Prisma because it would have to be the same values in the same tick